browser/resources/settings/brave_wallet_page/brave_wallet_page.ts

```diff
@@ -61,157 +108,163 @@ class SettingsBraveWalletPage extends SettingsBraveWalletPageBase {
-      { value: 'AUD' },
-      { value: 'BDT' },
-      { value: 'BHD' },
-      { value: 'BMD' },
-      { value: 'BRL' },
-      { value: 'CAD' },
-      { value: 'CHF' },
-      { value: 'CLP' },
-      { value: 'CZK' },
-      { value: 'DKK' },
-      { value: 'EUR' },
-      { value: 'GBP' },
-      { value: 'HKD' },
-      { value: 'HUF' },
-      { value: 'IDR' },
-      { value: 'ILS' },
-      { value: 'INR' },
-      { value: 'JPY' },
-      { value: 'KRW' },
-      { value: 'KWD' },
-      { value: 'LKR' },
-      { value: 'MMK' },
-      { value: 'MXN' },
-      { value: 'MYR' },
-      { value: 'NGN' },
-      { value: 'NOK' },
-      { value: 'NZD' },
-      { value: 'PHP' },
-      { value: 'PKR' },
-      { value: 'PLN' },
-      { value: 'RUB' },
-      { value: 'SAR' },
-      { value: 'SEK' },
-      { value: 'SGD' },
-      { value: 'THB' },
-      { value: 'TRY' },
-      { value: 'TWD' },
-      { value: 'UAH' },
-      { value: 'USD' },
-      { value: 'VEF' },
-      { value: 'VND' },
-      { value: 'ZAR' },
-      { value: 'XAG' },
-      { value: 'XAU' },
-      { value: 'XDR' }
+      { value: 'AED', name: 'AED' },
+      { value: 'ARS', name: 'ARS' },
+      { value: 'AUD', name: 'AUD' },
+      { value: 'BDT', name: 'BDT' },
+      { value: 'BHD', name: 'BHD' },
+      { value: 'BMD', name: 'BMD' },
+      { value: 'BRL', name: 'BRL' },
+      { value: 'CAD', name: 'CAD' },
+      { value: 'CHF', name: 'CHF' },
+      { value: 'CLP', name: 'CLP' },
+      { value: 'CZK', name: 'CZK' },
+      { value: 'DKK', name: 'DKK' },
+      { value: 'EUR', name: 'EUR' },
+      { value: 'GBP', name: 'GBP' },
+      { value: 'HKD', name: 'HKD' },
+      { value: 'HUF', name: 'HUF' },
+      { value: 'IDR', name: 'IDR' },
+      { value: 'ILS', name: 'ILS' },
+      { value: 'INR', name: 'INR' },
+      { value: 'JPY', name: 'JPY' },
+      { value: 'KRW', name: 'KRW' },
+      { value: 'KWD', name: 'KWD' },
+      { value: 'LKR', name: 'LKR' },
+      { value: 'MMK', name: 'MMK' },
+      { value: 'MXN', name: 'MXN' },
+      { value: 'MYR', name: 'MYR' },
+      { value: 'NGN', name: 'MGN' },
+      { value: 'NOK', name: 'NOK' },
+      { value: 'NZD', name: 'NZD' },
+      { value: 'PHP', name: 'PHP' },
+      { value: 'PKR', name: 'PKR' },
+      { value: 'PLN', name: 'PLN' },
+      { value: 'RUB', name: 'RUB' },
+      { value: 'SAR', name: 'SAR' },
+      { value: 'SEK', name: 'SEK' },
+      { value: 'SGD', name: 'SGD' },
+      { value: 'THB', name: 'THB' },
+      { value: 'TRY', name: 'TRY' },
+      { value: 'TWD', name: 'TWD' },
+      { value: 'UAH', name: 'UAH' },
+      { value: 'USD', name: 'USD' },
+      { value: 'VEF', name: 'VEF' },
+      { value: 'VND', name: 'VND' },
+      { value: 'ZAR', name: 'ZAR' },
+      { value: 'XAG', name: 'XAG' },
+      { value: 'XAU', name: 'XAU' },
+      { value: 'XDR', name: 'XDR' }
     ]
-    this.currency_list_.every((x) => x.name = x.value);
-  }
```

Comment:
maybe we should do an assert to make sure they all match?

Reply:
Good idea, done.